easybuild/easyconfigs/a/awscli/awscli-1.16.290-foss-2018b-Python-3.6.6.eb

```diff
@@ -9,16 +9,43 @@ description = 'Universal Command Line Environment for AWS'
 
 toolchain = {'name': 'foss', 'version': '2018b'}
 
-source_urls = [PYPI_SOURCE]
-sources = [SOURCE_TAR_GZ]
-checksums = ['6c5556e583924e9071195f773202bf77c538be72207510e6dc6baf85609f0955']
-
 dependencies = [
     ('Python', '3.6.6'),
 ]
 
 use_pip = True
-download_dep_fail = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('colorama', '0.4.1', {
+        'checksums': ['05eed71e2e327246ad6b38c540c4a3117230b19679b875190486ddd2d721422d'],
+    }),
+    ('PyYAML', '5.1.2', {
```

Comment:
@timeu Please use a `PyYAML` dependency instead (there's an easyconfig for `PyYAML` 3.13 with this toolchain)

The `PyYAML` we provide depends on the `libyaml` C library, which results in a lot faster YAML parsing than a Python-only installation...